src/analysis/AreaOfInterests.java

```diff
@@ -7,7 +7,7 @@
 import java.util.Set;
 
 public class AreaOfInterests {
-    final static String FIXATIONID_INDEX = "FPOGID";
+    final static String FIXATIONID_INDEX = "CNT"; //FPOGID
```

Comment:
@Immersor-git Could you explain why CNT is used over FPOGID? Not saying its wrong but I just want to understand the code better.